### DISCUSSION TITLE
N Largest Empty Circles

### DISCUSSION THREAD

**Discussion by micycle1** [locationtech/jts#857] [Ideas]
Could `org.locationtech.jts.algorithm.construct.LargestEmptyCircle` be straightforwardly adapted to find the N largest (ideally non-overlapping) empty circles?

**Comment by dr-jts** [comment]
Maybe...  That would require keeping a growing list of the centres of constructed largest circles, and including their (virtual) circles in the "distance-to" computation for further candidate centres.

Intriguing idea, for sure!
